sockeye/image_captioning/arguments.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
[cannot add a comment to empty file `image_captioning.py/__init__.py`] `__init__.py` is missing a license header.

Reply:
I will add the license header to `image_captioning.py/__init__.py`.